src/api/ingest/macroinvertebrate_serializer.py

```
@@ -0,0 +1,166 @@
+from .serializers import CollectRecordCSVSerializer
+
+__all__ = ["MacroInvertebrateCSVSerializer"]
+
+
+class MacroInvertebrateCSVSerializer(CollectRecordCSVSerializer):
+    protocol = MACROINVERTEBRATE_PROTOCOL
+    sample_unit = "beltinvert_transect"
+    observations_fields = ["data__obs_belt_inverts"]
+    additional_group_fields = CollectRecordCSVSerializer.additional_group_fields.copy()
+    additional_group_fields.append("data__beltinvert_transect__label")
+    composite_fields = {"data__sample_event__sample_date": ["year", "month", "day"]}
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)